docs/exchange-operators/rosetta/samples/index.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Code Samples
+description: Inspect integration code samples
+---
+
+:::info
+
+The code samples in this section are tested with Python 3.9
+
+:::
+
+The following series of guides demonstrate how to perform common Rosetta API use-cases. 
+
+All code samples are written in Python and require the `requests` library to be installed on your Python environment.
+
+This can be done with a following command:
+
+    pip3 install requests
+
+Also for using `pprint_json` function, you will need `jq` installed on your machine. On Linux machines, it is installed by default. For macOS, you can install it with brew package manager:
```

Comment:
I don't think this is the case. I have just checked a 22.04 LTS Ubuntu installation and it does not include `jq` by default. Arch Linux that I use does not ship with it either. It needs to be installed manually.

Reply:
removed jq